src/store/ChartTitleStore.js

```diff
@@ -116,4 +117,12 @@ export default class ChartTitleStore {
             }
         }
     }
+
+    @action.bound onMouseEnter() {
```

Comment:
@shayan-fs it doesn't need `@action-bound` here
```suggestion
    onMouseEnter = () => {
```